mlflow/store/db_migrations/versions/39d1c3be5f05_add_is_nan_constraint_for_metrics_tables_if_necessary.py

```diff
@@ -26,14 +26,20 @@ def upgrade():
     # to `True`
     with op.batch_alter_table("latest_metrics") as batch_op:
         batch_op.alter_column(
-            "is_nan", type_=sa.types.Boolean(create_constraint=True), nullable=False, default=False
+            "is_nan",
+            type_=sa.types.Boolean(create_constraint=True),
+            nullable=False,
+            server_default=False,
```

Comment:
I'm taking a look at doc for `alter_column`. Do we need to specify `existing_server_default` for MySQL?

cc @smurching 

https://alembic.sqlalchemy.org/en/latest/ops.html#alembic.operations.Operations.alter_column

Reply:
Let me check whether we set the default value for `is_nan`.

Reply:
We currently set the default value for the `metrics` table but not `latest_metrics`:

https://github.com/mlflow/mlflow/blob/28d998808dd26debcc10a643af12496dab27edac/tests/resources/db/latest_schema.sql#L87-L110